Source/WTF/wtf/ObjCRuntimeExtras.h

```diff
@@ -41,3 +45,48 @@ inline ReturnType wtfCallIMP(IMP implementation, id target, SEL selector, Argume
 }
 
 #endif // __cplusplus
+
+namespace WTF {
+
+inline MallocSpan<Method, SystemMalloc> class_copyMethodListSpan(Class cls)
```

Comment:
I understand that you are doing inlines here to avoid overhead and perhaps avoid a `.cpp` file. I’d be tempted to not do that because I don’t think function call overhead is significant compared to `malloc`/`free` and I try to avoid putting our code into headers unless it’s an important optimization.